lib/services/sub_product_from_warehouse.rb

```diff
@@ -0,0 +1,9 @@
+module Store
+  class SubProductFromWarehouse
```

Comment:
What does the `Sub` in `SubProductFromWarehouse` stands for? It's good habit to use full names